core/src/main/resources/application.yml

```diff
@@ -98,6 +98,8 @@ feast:
         authorizationUrl: http://localhost:8082
         subjectClaim: email
         
+    disableRestControllerAuth: false
```

Comment:
@SwampertX can we add a comment here to document what enabling and disabling the rest controller does (especially security implications ie rest controller is not authenticated.)